pkg/operator/test/containers.go

```diff
@@ -91,7 +91,7 @@ func (ct *ContainersTester) RunFullSuite(resourceExpectations ResourceLimitExpec
 func argEnvReferences(c *v1.Container) []string {
 	argRefSet := map[string]bool{}
 	for _, a := range c.Args {
-		argRefRegex, e := regexp.Compile("\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)")
+		argRefRegex, e := regexp.Compile(`\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)`)
```

Comment:
If you're changing the quotes, we shouldn't need to escape the backslashes anymore.

Reply:
I'm reverting changes as for a similar situation [here](https://github.com/rook/rook/pull/6260#discussion_r488733025) we were getting error in the build.

Reply:
Changing it to ```regexp.Compile(`\$\(([a-zA-Z][a-zA-Z0-9_]*)\)`)``` is  working, I'll change this.